tests/unit/drivers/querylang/test_querylang_drivers.py

```diff
@@ -87,14 +87,14 @@ def validate(req):
 
     f = (Flow().add(uses='DummySegmenter')
         .add(
-        uses='- !SortQL | {field: id, reverse: true, traversal_paths: [r, c, m]}'))
+        uses='- !SortQL | {field: id, reverse: true, traversal_paths: [\'c\']}'))
```

Comment:
This test should not pass. It tests, whether all three thing (root docs, matches and chunks) are reversed. If you only reverse chunks, it should fail.

Furthermore, I'd prefer having a plain `traversal_paths: [c]` instead of escaping characters. It makes the code easier to read.

Reply:
Sure, reverted `traversal_paths` to `[r, c, m]`